TheWarWithin/HunterBeastMastery.lua

```diff
@@ -786,15 +783,14 @@ spec:RegisterAuras( {
     howl_of_the_pack = {
         id = 462515,
         duration = 8.0,
-        max_stack = 1,
+        max_stack = 3,
     },
-    -- Talent: Your next Kill Shot is usable on any target, regardless of your target's current health.
-    -- https://wowhead.com/beta/spell=378215
-    hunters_prey = {
-        id = 378215,
-        duration = 15,
+    -- Talent: Your Kill Shot strikes $s1 more targets and deals $s3% more damage.
```

Comment:
?

Reply:
It's not actually a real trackable buff that matters for anything. If it ever gets referenced in the APL we can hack something together, but it's effectively just "current # of pets".